README.md

```diff
@@ -200,7 +202,7 @@ field is a free-form string that identifies the build and test runner.
 ### Robot Comments
 
 Robot comments are comments generated by static analysis tools. These are
-stored in the "refs/notes/devtools/analyses" ref, and annotate the revision.
+stored in the "refs/notes/git-review/robot" ref, and annotate the revision.
```

Comment:
scratch this. this should be the other way around i.e. talk about analyses, not "robot comments", and keep to `refs/notes/*/analyses`